english/marshmallow.md

```diff
@@ -280,7 +280,14 @@ Do you:
 [Eat the Marshmallow](eatmarshmallow/eatmarshmallow.md)
 
 [Escape into Willy Wonka Lego-World](lego-world/lego-world.md)
+<<<<<<< HEAD
+
+[Eat breakfast] (../english/breakfast/breakfast.md)
+
+||||||| merged common ancestors
+=======
 
 [The inception protocol](dream_in_dream/inception.md)
 
 [Pull out your proton pack!](ghost-buster/ghost-busted.md)
+>>>>>>> 1aead4324b63c9802e63bd83910671bd25627047
```

Comment:
It looks like this merge conflict didn't get resolved properly - the special merge conflict lines merge conflict will need to be removed.